### DISCUSSION TITLE
A little suggestion

### DISCUSSION THREAD

**Discussion by 873578156** [community/community#63480] [Mobile]
You can only edit files, but you cannot create files. I hope to add the ability to create files and modify file names.

**Comment by zMynxx** [comment]
Hey,
You can certainly add files, edit and rename.

* Add new files.
<img width="334" alt="Screenshot 2023-08-11 at 17 17 04" src="https://github.com/community/community/assets/124557045/63991247-d0e8-4e39-b62c-5f484e9e2fae">

* Click on the pen to edit the file.
<img width="245" alt="Screenshot 2023-08-11 at 17 18 35" src="https://github.com/community/community/assets/124557045/d3aeaf85-a25f-423e-b4fa-e7b016615356">

* Rename a file (after using the same edit bottonas mentioned above ^)
<img width="327" alt="Screenshot 2023-08-11 at 17 18 48" src="https://github.com/community/community/assets/124557045/25642e6b-ebed-4a7b-8535-d7317f57b785">

**Reply by 873578156** [reply]
nono，I'm talking about mobile apps.

**Comment by EdamAme-x** [comment]
It looks like it's possible.

You can press Add file and create it with create new file.

If you put a slash in the name part, the previous string will be automatically routed as a folder.

**Reply by 873578156** [reply]
nono，I'm talking about mobile apps.

**Reply by EdamAme-x** [reply]
> nono，I'm talking about mobile apps.

Sorry,

I thought it was a web version...

I'm actually writing from the mobile app.

I agree.

**Comment by migoa** [comment]
An option is to copy your files and put them on [termux](https://f-droid.org/packages/com.termux/) then commit and push them to github.
